src/lib/components/drs/DrsMain.js

```diff
@@ -0,0 +1,312 @@
+import '@fontsource/roboto';
+import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+import {
+  Switch,
+  Route,
+  useHistory
+} from "react-router-dom";
+import { Snackbar } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import { format } from 'date-fns';
+import DrsIndex from './pages/DrsIndex';
+import DrsObjectForm from './pages/DrsObjectForm';
+import DrsApiCaller from './utils/DrsApiCaller';
+import _ from 'lodash';
+import FormViewType from '../../model/common/FormViewType';
+
+const DrsMain = props => {
+
+  const emptyDrsObject = () => {
+    let date = new Date();
+    date.setSeconds(0, 0);
+    let emptyDrsObject = {
+      id: '',
+      description: '',
+      created_time: format(date, "yyyy-MM-dd'T'HH:mm:ss'Z'"),
+      mime_type: '',
+      name: '',
+      size: '',
+      updated_time: format(date, "yyyy-MM-dd'T'HH:mm:ss'Z'"),
```

Comment:
Timestamps are now handled correctly when the user selects a time, however the issue persists with the default time (i.e. the current time which is auto-populated when the NEW page is loaded). If the user does not update the time from what is set by default when the page loads, the timestamp displayed in the UI is offset from the actual current time (for example, in EST local time, the value displayed in the UI is 4 hours behind the current time). However, the value submitted to the API is correctly converted to UTC. In this case the time conversion is correct, however the default time shown in the UI is offset from the actual current time (until the user manually enters a time, in which case it is displayed and converted correctly).